source_test.yaml

```diff
@@ -44,6 +44,7 @@
 - name: 'android'
   versions_from_repo: False
   type: 1
+  ignore_patterns: ['^(?!A-).*$']
```

Comment:
I don't think this one is correct. 

There may be other cases for us to consider. They also export PUB-A-*.json for example.

Reply:
Good catch, thanks. Corrected in 203fec69